packages/docusaurus/bin/docusaurus.js

```diff
@@ -39,15 +39,15 @@ cli
   .description('Build website')
   .option(
     '--bundle-analyzer',
-    'Visualize size of webpack output files with an interactive zoomable treemap (default = false)',
+    'Visualize size of webpack output files with an interactive zoomable treemap (default: disabled)',
```

Comment:
I think we should retain the boolean values here as it's much easier for the user to know what value to use. Could you change them back?